microsoft-edge/visual-studio/index.md

```diff
@@ -115,13 +112,13 @@ From the **Attach to Process** dialog, set **Connection type** to **Chrome devto
    Configure the **Attach to Process** dialog in Visual Studio  
 :::image-end:::  
 
-Choose **Select...** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, choose the **Refresh** button.  Choose the tab you want to debug and choose **Attach**.  
+Select **Select** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, select the **Refresh** button.  Choose the tab you want to debug and select **Attach**.  
 
 The Visual Studio debugger is now attached to Microsoft Edge.  You may pause the running of JavaScript, set breakpoints, and review `console.log()` statements directly in the Debug Output window in Visual Studio.  
```

Comment:
Global: check 'can' and 'may'.  Change 'may' to 'can' in most places.  Per MWSG's [can, may](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33430) article: "When ability is what you need to express, it's OK to use can to describe actions or tasks that the reader or program is able to do. Use might to express possibility. **Don't use may**, which might be interpreted as providing permission."